instabot/bot/bot_get.py

```diff
@@ -195,15 +195,15 @@ def get_comment_likers(self, comment_id):
     if "users" not in self.api.last_json:
         self.logger.info("Comment with %s not found." % comment_id)
         return []
-    return list(map(lambda user: str(user['pk']), self.api.last_json["users"]))
+    return list([str(user['pk']) for user in self.api.last_json["users"]])
```

Comment:
Again, this doesn't seem necessary but if you are going to switch it over to a list comprehension, why would you wrap it with `list(...)` afterwards?

Reply:
How is this resolved?